.github/workflows/build_linux_pytorch_wheels.yml

```
@@ -88,6 +90,18 @@ jobs:
       - name: Upload wheels to S3
         if: ${{ github.repository_owner == 'ROCm' }}
         run: |
-          # These wheels share the same S3 directory as rocm-sdk to simplify --find-links usage
-          aws s3 cp ${{ env.PACKAGE_DIST_DIR }}/ s3://therock-nightly-python/${{ inputs.AMDGPU_FAMILIES }}/ \
+          aws s3 cp ${{ env.PACKAGE_DIST_DIR }}/ \
+            s3://${{ env.S3_PYTHON_BUCKET }}/${{ inputs.AMDGPU_FAMILIES }}/${{ env.PYTHON_TAG }}/ \
```

```suggestion
            s3://${{ env.S3_PYTHON_BUCKET }}/${{ inputs.AMDGPU_FAMILIES }}/ \
```
Why not having the wheels for different Python versions side-by-side?

---

+1, pip is built to choose wheels based on python version. What it _isn't_ built to do is choose wheels based on which GPU or GPU drivers you have installed. That's why we separate GPU families into separate buckets and index pages.

---

Make sense working on it.